app/src/js/nav.js

```diff
@@ -4,24 +4,40 @@ export function createNavigation() {
     const menu = document.querySelector(".usa-nav");
     const menuIcon = document.querySelector(".usa-accordion__button svg use");
 
-    if (toggleButton && closeButton && menu) {
+    function toggleMenu(e) {
+        e.preventDefault();
+        const isOpen = menu.classList.toggle("is-visible");
 
-        toggleButton.addEventListener("click", () => {
-            const isOpen = menu.classList.toggle("is-hidden");
+        toggleButton.setAttribute("aria-expanded", isOpen ? "true" : "false");
+        menuIcon.setAttribute(
+            "href",
+            isOpen ? "/assets/img/sprite.svg#close" : "/assets/img/sprite.svg#menu"
```

Comment:
Toggling looks a bit weird here when closing the menu. It quickly changes from the X icon back to the menu one. Maybe when the menu is visible, we can have the close/X icon  at the top of the menu or above it, then when closed/clicked on, it disappears?
https://github.com/user-attachments/assets/c4e99266-3606-4d5e-a47b-d62a291b5f17

Reply:
Yes! I have two issues with it, that and the underline under the menu. I was going to look at it again after OSAB, but I thought to move on to the graphs for now was more pertinent.